x-pack/plugins/observability/public/components/shared/alert_search_bar/alert_search_bar.tsx

```diff
@@ -6,16 +6,15 @@
  */
 
 import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
-
 import React, { useCallback, useEffect } from 'react';
+
 import { i18n } from '@kbn/i18n';
 import { Query } from '@kbn/es-query';
-import { useKibana } from '../../../utils/kibana_react';
-import { observabilityAlertFeatureIds } from '../../../config';
-import { ObservabilityAppServices } from '../../../application/types';
+import { useServices } from './services';
```

Comment:
What benefit does it have to introduce a new `useServices` which just exports `useKibana().services` ?

Reply:
Since this component will be used in another plugin, we don't know if they have all the related dependencies defined in their plugin as this shared component needs.
So I used a similar pattern as mentioned in Clint's presentation to make these dependencies explicit and the consumer should make sure to pass the related services to the `ObservabilityAlertSearchBarProvider` that I defined here.